hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/SubscriptionsEndpoint.java

```diff
@@ -154,6 +155,15 @@ public SubscriptionMetrics getMetrics(@PathParam("topicName") String qualifiedTo
         return subscriptionService.getSubscriptionMetrics(fromQualifiedName(qualifiedTopicName), subscriptionName);
     }
 
+    @GET
+    @Produces(APPLICATION_JSON)
+    @Path("/{subscriptionName}/metrics/zookeeper")
+    @ApiOperation(value = "Get subscription zookeeper metrics", response = SubscriptionZookeeperMetrics.class, httpMethod = HttpMethod.GET)
+    public SubscriptionZookeeperMetrics getZookeeperMetrics(@PathParam("topicName") String qualifiedTopicName,
```

Comment:
I suggest not to expose the implementation details (`zookeeper`) in the API. In my opinion it would be better to replace `zookeeper` with, for example `persistent`.